.vscode/settings.json

```
@@ -70,5 +70,11 @@
       "wordpress",
       "workdir",
       "yaml's"
-    ]
+    ],
+    "cSpell.ignoreWords": [
+      "file"
+    ],
+    "files.autoSave": "onFocusChange",
+    "files.autoSaveDelay": 100,
+    "editor.fontSize": 13
```

Looks like you accidentally add your global settings to local repo.
Please, remove it
```suggestion
    ]
```